tmc-langs-framework/src/test/java/fi/helsinki/cs/tmc/langs/ClassPathTest.java

```diff
@@ -0,0 +1,74 @@
+
+package fi.helsinki.cs.tmc.langs;
+
+import com.google.common.base.Throwables;
+import java.io.File;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ClassPathTest {
+    
+    private ClassPath cp;
+    
+    @Before
+    public void setUp() {
+        cp = new ClassPath(getPath("ant_project"));
```

Comment:
Changed the constructor to the suggested one, only reason was that currently we had only one use for the ClassPath.